src/login/LoginPage.jsx

```diff
@@ -1,395 +1,318 @@
-        ) : (
-          <>
-            {(isEnterpriseLoginDisabled && isInstitutionAuthActive) && (
-              <RenderInstitutionButton
-                onSubmitHandler={this.props.handleInstitutionLogin}
-                buttonTitle={intl.formatMessage(messages['institution.login.button'])}
-              />
-            )}
-            {isSocialAuthActive && (
-              <div className="row m-0">
-                <SocialAuthProviders socialAuthProviders={providers} />
-              </div>
-            )}
-          </>
-        )}
-      </>
+      <InstitutionLogistration
+        secondaryProviders={secondaryProviders}
+        headingTitle={formatMessage(messages['institution.login.page.title'])}
+      />
     );
   }
-
-  renderForm(
-    currentProvider,
-    providers,
-    secondaryProviders,
-    thirdPartyAuthContext,
-    thirdPartyAuthApiStatus,
-    submitState,
-    intl,
-  ) {
-    const activationMsgType = getActivationStatus();
-    if (this.props.institutionLogin) {
-      return (
-        <InstitutionLogistration
-          secondaryProviders={thirdPartyAuthContext.secondaryProviders}
-          headingTitle={intl.formatMessage(messages['institution.login.page.title'])}
+  return (
+    <>
+      <Helmet>
+        <title>{formatMessage(messages['login.page.title'], { siteName: getConfig().SITE_NAME })}</title>
+      </Helmet>
+      <RedirectLogistration
+        success={loginResult.success}
+        redirectUrl={loginResult.redirectUrl}
+        finishAuthUrl={finishAuthUrl}
+      />
+      <div className="mw-xs mt-3 mb-2">
+        <LoginFailureMessage
+          errorCode={errorCode.type}
+          failureCount={errorCode.count}
+          context={errorCode.context}
         />
-      );
-    }
-    const tpaAuthenticationError = {};
-    if (thirdPartyAuthContext.errorMessage) {
-      tpaAuthenticationError.context = {
-        errorMessage: thirdPartyAuthContext.errorMessage,
-      };
-      tpaAuthenticationError.errorCode = TPA_AUTHENTICATION_FAILURE;
-    }
-
-    return (
-      <>
-        <Helmet>
-          <title>{intl.formatMessage(messages['login.page.title'],
-            { siteName: getConfig().SITE_NAME })}
-          </title>
-        </Helmet>
-        <RedirectLogistration
-          success={this.props.loginResult.success}
-          redirectUrl={this.props.loginResult.redirectUrl}
-          finishAuthUrl={thirdPartyAuthContext.finishAuthUrl}
+        <ThirdPartyAuthAlert
+          currentProvider={currentProvider}
+          platformName={platformName}
+        />
+        <AccountActivationMessage
+          messageType={activationMsgType}
         />
-        <div className="mw-xs mt-3">
-          <ThirdPartyAuthAlert
-            currentProvider={thirdPartyAuthContext.currentProvider}
-            platformName={thirdPartyAuthContext.platformName}
+        {showResetPasswordSuccessBanner && <ResetPasswordSuccess />}
```

Comment:
in prev implementation, this was behind two checks, the `showResetPasswordSuccessBanner` and the `!this.props.loginError` but I could not find the second check in refactored code?

Reply:
This was an additional check that we don't need. We already have code in the handleSubmit that dismisses the password reset banner once the "Sign in" button is clicked. There will never be a scenario when we have both password reset banner and some error message banner.